pkg/app/execontext.go

```diff
@@ -197,13 +229,41 @@ func (ref *Output) Info(infoType string, params ...OutVars) {
 		}
 	}
 
-	fmt.Printf("cmd=%s info=%s%s%s\n", ref.CmdName, itcolor(infoType), sep, data)
+	switch ref.JSONFlag {
```

Comment:
This is similar to the State call (we have the `params` slice)

Reply:
Tried to fix this too, do review. thanks